WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,8 +1,9 @@
-- Feature image settings in Latest Post block
-- HTML/Visual mode switch notices
-- Full Embed block preview
-- Locale Picker visuals
-- Longer uploads with standalone VideoPress
-- Up-to-date Atomic site previews
-- Media Picker talkback media types
-- Signup magic link URL
+* [*] Pages List: Added "Copy Link" button to context menu [https://github.com/wordpress-mobile/WordPress-Android/pull/16321]
+* [*] Editor: Improved editor performance by reducing memory usage [https://github.com/wordpress-mobile/WordPress-Android/pull/16490]
+* [***] Enable Site Domains purchases for Jetpack app only [https://github.com/wordpress-mobile/WordPress-Android/pull/16417]
```

Comment:
Removed the "Jetpack App:" prefix and kept this line only in the Jetpack release notes draft while removing it from the WordPress ones. \cc @leandroalonso

Reply:
Actually let's also remove the suffix mention as well…
```suggestion
* [***] Enable Site Domains purchases [https://github.com/wordpress-mobile/WordPress-Android/pull/16417]
```

Reply:
Tbh given how very explicit that entry was that it was Jetpack only (thanks @ravishanker for making that clear 💙 ), I don't think I really need to wait for your confidence check on that one 😄 
_(I only made that comment as a matter of habit of inline-commenting on any of those diff between WP and JP release notes to document each of them… but that one is pretty clear and explicit 😅 so, sorry to have bothered and ping'd you)_